Solutions/Problem_Statement_48_solution.java

```diff
@@ -0,0 +1,45 @@
+/*Write a program to find line of best fit for any 5 points entered by the user.
+(take input of x and y coordinates of each point from user).*/
+import java.util.Scanner;
+public class Problem_Statement_48_solution {
+    public static void main(String[] args) {
+        Scanner sc=new Scanner(System.in);
+        int n = 5;//Only for 5 points given in problem
+        int x[]=new int[n];//Array of elements of x coordinate
+        int y[]=new int[n];//Array of y coordinate
+

```

Comment:
Hello @komalvhanmane,
Your output is correct but the program isn't user friendly, like It only shows a blank screen and I don't know what I was supposed to give as input. Could you please do the changes so that the output specifies what data is expected from the user!
You can do it this way,

Output:

Enter the coordinates of 1st point:
(user enters coordinates of 1st point)
Enter the coordinates of 2nd point:
and so on...

Reply:
Okay

Reply:
Done , Can you Please Check now.